helx/charts/roger/values.yaml

```diff
@@ -0,0 +1,213 @@
+redis:
+  image:
+    repository: redislabs/redisgraph
+    tag: 2.2.13
+  redis:
+    command: "redis-server"
+  clusterDomain: "blackbalsam-cluster"
+  cluster:
+    slaveCount: 1
+  usePassword: false
+  master:
+    command: ""
+    readinessProbe:
+      enabled: false
+    livenessProbe:
+      enabled: false
+    extraFlags:
+      - "--loadmodule /usr/lib/redis/modules/redisgraph.so"
+  slave:
+    command: ""
+    readinessProbe:
+      enabled: false
+    livenessProbe:
+      enabled: false
+    extraFlags:
+      - "--loadmodule /usr/lib/redis/modules/redisgraph.so"
+
+
+airflow:
+  #
+  # NOTE:
+  # - This is intended to be a `custom-values.yaml` starting point for non-production deployment (like minikube)
+
+  # External Dependencies:
+  # - A PUBLIC git repo for DAGs: ssh://git@repo.example.com:my-airflow-dags.git
+  #
+
+  ###################################
+  # Airflow - Common Configs
+  ###################################
+  airflow:
+    ## the airflow executor type to use
+    ##
+    image:
+      repository: renciorg/apache-airflow-1.10.14-python-3.8-git
+      tag: latest
+    #  executor: CeleryExecutor
+    executor: KubernetesExecutor
+
+    ## the fernet key used to encrypt the connections in the database
+    ##
+    fernetKey: "7T512UXSSmBOkpWimFHIVb8jK6lfmSAvx4mO6Arehnc="
```

Comment:
What's this used for and should it be here?  Set it to a random value on deployment?

Reply:
Fernet key is now auto-generated (https://github.com/helxplatform/devops/blob/roger-helm/helx/charts/roger/templates/airflow-config-secrets.yaml ) and used  [here](https://github.com/helxplatform/devops/blob/bfe61fb840ecb10d27422ca0639a4596f195a20a/helx/charts/roger/values.yaml#L100)